manifests/repo/redhat.pp

```diff
@@ -7,18 +7,18 @@
   }
 
   yumrepo { 'Zend':
-    baseurl  => "http://repos.zend.com/zend-server/${zendserver::zend_server_version}/rpm/${::architecture}",
```

Comment:
I think that it is a single digit only, but 7.0 is also not in rpm_apache2.4 so I think that someone could not install 7.